docs/java/java-debugging.md

```diff
@@ -154,21 +157,26 @@ Below are all the configurations available for `Launch` and `Attach`, for more i
 
 ### Launch
 
-- `mainClass` (required) - The fully qualified class name (for example [java module name/]com.xyz.MainApp) or the java file path of the program entry.
-- `args` - The command-line arguments passed to the program. Use `"${command:SpecifyProgramArgs}"` to prompt for program arguments. It accepts a string or an array of string.
+- `mainClass` (required) - The fully qualified class name (e.g. [java module name/]com.xyz.MainApp) or the java file path of the program entry.
```

Comment:
It looks like you might not be current with the latest version since you are reverting editing changes.
For example:
we don't use e.g.
command-line arguments is correct (with a hyphen)
Java should be capitalized
The feature is CodeLens

Reply:
You're right, I just copied over from another source with new configurations we support now.